src/test/java/io/vincenzopalazzo/placeholder/DemoFrame.java

```diff
@@ -63,15 +65,24 @@ public void initComponent() {
     Icon icon =
         MaterialImageFactory.getInstance()
             .getImage(MaterialIconFont.LOCATION_SEARCHING, 16, MaterialColors.COSMO_DARK_GRAY);
+    Icon menu =
+        MaterialImageFactory.getInstance()
+            .getImage(MaterialIconFont.MORE_VERT, 22, MaterialColors.COSMO_DARK_GRAY);
     JTextField textField = new JTextField();
     textField.setPreferredSize(new Dimension(100, 20));
-    textFieldPlaceholder = new JMarsSearch(textField, icon);
+    textFieldPlaceholder = new JMarsSearch(textField, icon, menu);
     icon =
         MaterialImageFactory.getInstance()
             .getImage(MaterialIconFont.SEARCH, 20, MaterialColors.COSMO_DARK_GRAY);
     textFieldPlaceholder
+        .addMenuActions(
```

Comment:
@zanderson9 To add the action in the button.

Alert! The order of the call is important!